ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/DayHeader.kt

```diff
@@ -53,26 +56,46 @@ fun DayHeader(
         verticalAlignment = Alignment.CenterVertically,
         horizontalArrangement = Arrangement.spacedBy(16.dp),
     ) {
-        Column(
-            horizontalAlignment = Alignment.CenterHorizontally,
-            verticalArrangement = Arrangement.spacedBy((-10).dp)
-        ) {
-            StyledText(
-                month,
-                style = KotlinConfTheme.typography.text2,
-                color = white60,
-            )
-            StyledText(
-                day,
-                style = DayDateStyle,
-                color = KotlinConfTheme.colors.primaryTextInverted,
-                modifier = Modifier.widthIn(min = 72.dp),
-            )
-            StyledText(
-                "",
-                style = KotlinConfTheme.typography.text2,
-                color = Color.Transparent,
-            )
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.spacedBy((-10).dp)
+            ) {
+                StyledText(
+                    month,
+                    style = KotlinConfTheme.typography.text2,
+                    color = white60,
+                )
+                StyledText(
+                    day,
+                    style = DayDateStyle,
+                    color = KotlinConfTheme.colors.primaryTextInverted,
+                    modifier = Modifier.widthIn(min = 72.dp),
+                )
+                StyledText(
+                    "",
+                    style = KotlinConfTheme.typography.text2,
+                    color = Color.Transparent,
+                )
+            }
+            if (day2.isNotEmpty()) {
+                StyledText(
+                    "",
+                    style = KotlinConfTheme.typography.text2,
+                    color = white60,
+                )
```

Comment:
I believe these empty texts can just be omitted from the second day as the first day already provides the spacing, and we can just center this vertically